packages/jbrowse-plugin-apollo/src/ApolloInternetAccount/model.ts

```diff
@@ -389,3 +422,127 @@ export type ApolloInternetAccountStateModel = ReturnType<
 >
 export type ApolloInternetAccountModel =
   Instance<ApolloInternetAccountStateModel>
+
+/**
+ * Get server timestamp and save it into session storage
+ * @param apolloInternetAccount - apollo internet account
+ * @returns
+ */
+async function getAndSetServerTime(session: ApolloSession) {
```

Comment:
This should get the sequence number of the most recent change instead of using a timestamp.

Reply:
Fixed (wrong branch was pushed)